sdk/digitaltwins/azure-digitaltwins-core/README.md

```diff
@@ -11,6 +11,37 @@ The complete Microsoft Azure SDK can be downloaded from the [Microsoft Azure dow
 For the best development experience, developers should use the official Microsoft Maven packages for libraries. Maven packages are regularly updated with new functionality and hotfixes.
 
 ### Include the Package
+#### Include the BOM file
+
+Please include the azure-sdk-bom to your project to take dependency on the latest GA version of the library.
+
+```xml
+<dependencyManagement>
+    <dependencies>
+        <dependency>
+            <groupId>com.azure</groupId>
+            <artifactId>azure-sdk-bom</artifactId>
+            <version>1.0.3</version>
```

Comment:
What keeps this version up to date?

Reply:
I have updated the message to not have the version and have the readme link instead,

Here is the thought process behind it -
We can build tooling to keep this version current in the repo - however once a package is released we can't update its readme.